chromium_src/chrome/browser/ui/views/tabs/alert_indicator_button.cc

```diff
@@ -0,0 +1,12 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "chrome/browser/ui/views/tabs/tab.h"
+
+#define GetWidthOfLargestSelectableRegion GetWidthOfLargestSelectableRegionBrave
```

Comment:
also I don't think we we should keep the original version of this function, why not replace it fully with our impl and keep the same name?

Reply:
At first, I created `BraveTab` subclass and replace it with `Tab` in `TabStrip` class.
```
  Tab* tab = tab_container_->AddTab(
      std::make_unique<Tab>(this), model_index,
      pinned ? TabPinned::kPinned : TabPinned::kUnpinned);
```
However, `Tab*` is also used from many methods as a parameter type and
it gave compile failure due to `BraveTab*` as a changed parameter type.
I thought it was not easy w/o patching(replacing `Tab` directly).
So, I did this here.

Reply:
I thought of something like this:
https://github.com/brave/brave-core/blob/fc3931117f4e86f57b96f2a6fafc60fb73ff3508/chromium_src/chrome/browser/lookalikes/lookalike_url_navigation_throttle.h#L9-L12
https://github.com/brave/brave-core/blob/fc3931117f4e86f57b96f2a6fafc60fb73ff3508/chromium_src/chrome/browser/lookalikes/lookalike_url_navigation_throttle.cc#L8-L16

Reply:
A ha! Updated. Thanks 👍🏼